pkg/knode-manager/adapters/interface.go

```diff
@@ -29,3 +29,9 @@ type NodeHandler interface {
 
 	Configure(context.Context, *corev1.Node)
 }
+
+type SecretHandler interface {
+	Update(old, new *corev1.Secret) (*corev1.Secret, error)
```

Comment:
Update and Delete method can be unified with other handlers ，eg PodHandler

Reply:
I got it